tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -368,38 +367,35 @@ export async function assertIdentity(
 
 export function assertWorkerError(
     context: IntegrationTestContext,
-    requestIdentifier: string,
     check: (returnValue: StfError) => void,
     returnValue: WorkerRpcReturnValue
 ) {
-    const errDecodedRes = context.api.createType(
-        'TrustedOperationResponse',
-        returnValue.value
-    ) as unknown as TrustedOperationResponse;
-    assert.equal(u8aToHex(errDecodedRes.req_ext_hash), requestIdentifier);
-    const errValueDecoded = context.api.createType('StfError', errDecodedRes.value) as unknown as StfError;
+    const errValueDecoded = context.api.createType('StfError', returnValue.value) as unknown as StfError;
     check(errValueDecoded);
 }
 
 export function assertIdentityLinkedResult(
     context: IntegrationTestContext,
-    requestIdentifier: string,
     expectedIdentity: LitentryPrimitivesIdentity,
     returnValue: WorkerRpcReturnValue
 ) {
-    const decodedRes = context.api.createType(
-        'TrustedOperationResponse',
-        returnValue.value
-    ) as unknown as TrustedOperationResponse;
-    assert.equal(decodedRes.req_ext_hash.toHex(), requestIdentifier);
-
     const decodedLinkResult = context.api.createType(
         'LinkIdentityResult',
-        decodedRes.value
+        returnValue.value
     ) as unknown as LinkIdentityResult;
 
     assert.isNotNull(decodedLinkResult.id_graph);
 }
+
+export function assertSetUserShieldingKeyResult(context: IntegrationTestContext, returnValue: WorkerRpcReturnValue) {
+    const decodedLinkResult = context.api.createType(
+        'SetUserShieldingKeyResult',
+        returnValue.value
+    ) as unknown as SetUserShieldingKeyResult;
+
+    assert.isNotNull(decodedLinkResult.id_graph);
```

Comment:
Just checking for `not null` here might not be sufficient. What we might be looking for is a check against the expected content inside the idgraph. If you're unsure how to do this here, you can add a TODO for now, and we can come back to it later to fill in the details.Thanks!

Reply:
The same issue applies to assertIdentityLinkedResult as well.:)

Reply:
True, I will try to add more precise assertions in this PR.

Reply:
You can use [assertInitialIdGraphCreated](https://github.com/litentry/litentry-parachain/blob/remove-req-ext-hash/tee-worker/ts-tests/integration-tests/common/utils/assertion.ts#L59) as a reference.

Reply:
Thanks 👍🏼

Reply:
I've added assertions